pkg/dockerregistry/server/auth.go

```diff
@@ -53,7 +53,7 @@ const (
 // RegistryClient encapsulates getting access to the OpenShift API.
 type RegistryClient interface {
 	// Clients return the authenticated clients to use with the server.
-	Clients() (client.Interface, kclientset.Interface, error)
+	Clients() (client.Interface, kcoreclient.LimitRangesGetter, error)
```

Comment:
Changed back to kclientset.Interface.
I'm not sure which solution is better: `kClient != nil` [1] check or unused `fake.NewSimpleClientset()`?
[1]: https://github.com/openshift/origin/pull/13366/files#diff-1a18cb8c0d7684cefcef809b9d01aac3R115